acra/security-controls/encryption/_index.md

```diff
@@ -2,3 +2,129 @@
 title: Encryption
 bookCollapseSection: true
 ---
+
+# Encryption
+
+Data encryption may be the main feature provided by Acra,
+even though there are plenty of other useful tools, such as SQL firewall.
+This page describes how to configure it and what you will get as a result.
+
+## Behavior
+
+AcraServer works as a proxy between database client and the database itself.
+When a column is configured to be encrypted, AcraServer will transparently encrypt/decrypt its value.
+However, there are some caveats:
+* In simplest scenario, AcraServer clients won't see any difference: they `INSERT` plaintext,
+  they get plaintext in `SELECT` queries, just like with direct connection to database.
+  At the same time, the data in the database itself will be stored in encrypted form.
+  And if the attacker has full access to the database (but not to the AcraServer),
+  he won't be able to magically bypass the encryption, no keys means no data.
+* Every AcraServer client has own unique identifier (usually derived during TLS or Secure Session handshake),
+  and a keystore associated with that identifier.
```

Comment:
```suggestion
  and a keys associated with that identifier.
```
we associate set of keys associated with that identifier and store it in keystore. It's not a keystore. In keystore v2 we use term `keyrings` that store group of keys in relation with their purpose